src/vivarium_public_health/risks/base_risk.py

```diff
@@ -272,11 +274,16 @@ def get_exposure_pipeline(self, builder: Builder) -> Pipeline:
     ########################
 
     def on_initialize_simulants(self, pop_data: SimulantData) -> None:
-        self.population_view.update(
-            pd.Series(
-                self.randomness.get_draw(pop_data.index), name=self.propensity_column_name
-            )
-        )
+        propensity_values = self.randomness.get_draw(pop_data.index)
+        exposure_values = self.exposure_distribution.ppf(propensity_values)
+        propensity_col = pd.Series(propensity_values, name=self.propensity_column_name)
+        exposure_col = pd.Series(exposure_values, name=self.exposure_column_name)
+        self.population_view.update(pd.concat([propensity_col, exposure_col], axis=1))
```

Comment:
Nit: this can be more concisely done like this:

```python
df = pd.DataFrame(
    {
        self.propensity_column_name: self.randomness.get_draw(pop_data.index),
        self.exposure_column_name: self.exposure_distribution.ppf(propensity_values),
    }
)
```